tests/test_i18n.py

```diff
@@ -17,3 +19,38 @@ def test_i18n():
         humanize.i18n.deactivate()
         assert humanize.naturaltime(three_seconds) == "3 seconds ago"
         assert humanize.ordinal(5) == "5th"
+
+
+def test_i18n_default_locale_path():
```

Comment:
Please can you split this into three separate functions, one for defined, one for `None` and one for not defined?

This'll ensure one thing is tested at a time; that the later parts of the test does not depend on the earlier parts.